int16 frontLeft
int16 frontCenter
int16 frontRight
int16 right
int16 rearRight
int16 rearCenter
int16 rearLeft
int16 left
int32 ping

